apps/modern-interviewer/src/styles/tailwind.css

```
@@ -0,0 +1,31 @@
+@import "@codaco/tailwind-config/fresco.css";
+
+/* Source the @codaco/fresco-ui and @codaco/interview compiled outputs so
+   Tailwind discovers utility classes referenced inside those packages.
+   Without this, those components render unstyled. */
+@source "../../../../node_modules/@codaco/fresco-ui/dist/**/*.js";
```

Stylelint (17.11.0)

[error] Unexpected unknown at-rule "@source" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)